patterns/project-description.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Title: Project Description
+ * Slug: twentytwentyfour/project-description
+ * Categories: portfolio, media
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"6rem","right":"6rem","bottom":"6rem","left":"6rem"},"margin":{"top":"0px","bottom":"0px"}}},"backgroundColor":"accent-2","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-accent-2-background-color has-background" style="margin-top:0px;margin-bottom:0px;padding-top:6rem;padding-right:6rem;padding-bottom:6rem;padding-left:6rem"><!-- wp:columns {"align":"wide","style":{"spacing":{"margin":{"bottom":"6rem"}}}} -->
+<div class="wp-block-columns alignwide" style="margin-bottom:6rem"><!-- wp:column {"width":"50%"} -->
+<div class="wp-block-column" style="flex-basis:50%"><!-- wp:paragraph {"align":"left","fontSize":"small"} -->
+<p class="has-text-align-left has-small-font-size"><?php echo esc_html__( 'Piva Hotel - Overview', 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"50%"} -->
+<div class="wp-block-column" style="flex-basis:50%"><!-- wp:paragraph {"style":{"layout":{"selfStretch":"fixed","flexSize":"50%"},"typography":{"fontSize":"2.38rem","lineHeight":"1.3","fontStyle":"normal","fontWeight":"400"}},"fontFamily":"cardo"} -->
+<p class="has-cardo-font-family" style="font-size:2.38rem;font-style:normal;font-weight:400;line-height:1.3"><?php echo esc_html__( 'Sessione fotografica per il Piva Arts &amp; Crafts Hotel vicino al lago Piva, in Montenegro. Un lago artificiale è stato costruito nel 1975, dividendo il Piva Kanyon per costruire una diga Mratinje.', 'twentytwentyfour' ); ?></p>
```

Comment:
We need to make sure that all sample text is in English and uses a translator context (esc_html_x() ).